src/core/transposition.h

```diff
@@ -109,7 +109,7 @@ class TranspositionTable {
     constexpr static void prefetch([[maybe_unused]] uint64_t key)
     {
         assert(s_tableSize > 0);
-        __builtin_prefetch(&s_table[key % s_tableSize]);
```

Comment:
To avoid statistical bias, at line 97 why don't we just make `s_tableSize` a power of two? :)

Reply:
because it's scaled based on UCI input. We can't guarantee that the user always inputs a power of two size.
We can ofc scale it to a power of two, but we might miss out on potentially huge amounts of TT storage instead which will most likely cause a bigger loss than win (eg downscale to 128 if eg 200 was used).

Most tournaments are aware of this and always use power of 2 sizes :)